modules/ivc_champva/app/services/ivc_champva/attachments.rb

```diff
@@ -15,7 +15,13 @@ def handle_attachments(file_path)
                         else
                           "#{uuid}_#{form_id}_supporting_doc-#{index}.pdf"
                         end
-        new_file_path = File.join(File.dirname(attachment), new_file_name)
+
+        new_file_path = if Flipper.enabled?(:champva_pdf_decrypt, @current_user)
+                          File.join('tmp', new_file_name)
+                        else
+                          File.join(File.dirname(attachment), new_file_name)
+                        end
+
```

Comment:
Updated here - encrypted PDF file paths were being prepended with `clamav_tmp`, which meant we weren't attaching them to our final PDF conglomerate.

This overrides that prefix with our directory `tmp`.